index.js

```diff
@@ -39,6 +39,11 @@ const make = (files, opts) => {
 			continue;
 		}
 
+		if (['vim', 'neovim'].indexOf(editor.id) !== -1) {
+			args.push(`"+call cursor(${parsed.line}, ${parsed.column})"`, parsed.file);
```

Comment:
You are totally correct, it works without the quotes. I hit an issue though, see other comment